### PR TITLE
fix: replace puzzle icon with the webapp variant

### PR DESCRIPTION
**Pull request by Spring3**
# Purpose of PR

Update puzzle icon to align it with the one used in the web app. Got this one from @fabe

## PR Checklist

- [ ] I have read the relevant `readme.md` file(s)
- [ ] All commits follow our [Git commit message convention](https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components#commits)
- [ ] Tests are added/updated/not required
- [ ] Tests are passing
- [ ] Storybook stories are added/updated/not required
- [ ] Usage notes are added/updated/not required
- [ ] Has been tested based on [Contentful's browser support](https://www.contentful.com/faq/about-contentful/#which-browsers-does-contentful-support)
- [ ] Doesn't contain any sensitive information

### CODE REVIEW

File: packages/forma-36-react-components/src/components/Icon/svg/Puzzle.tsx [hunk 1]
```diff
@@ -2,9 +2,8 @@ import * as React from 'react';
...
...009c.98820242 0 1.93878497.3644982 2.6719907 1.0190548l.16526074.1561666c.75248132.7524041 1.17524856 1.773083 1.17524847 2.8375353-.00072802 1.0636788-.42377298 2.0834966-1.17621016 2.8355036-.69867083.6981689-1.62822477 1.1122206-2.60910371 1.1678088l-.22718604.0064309-.76079-.0004.00079 3.3754c0 .6869835.53182749 1.2589531 1.20268154 1.3238665l.12731846.0061335 3.37421-.0004.00079009-.7623567c.00067603-.9875212.36534828-1.9372872 1.01963952-2.6699341l.15609759-.1651365c.75215585-.7518028 1.77199216-1.1743418 2.8355895-1.1746727 1.0637192.0003309 2.0837533.4230399 2.8359483 1.175235.6984206.6984205 1.1128294 1.6278448 1.1687323 2.6087661l.0065027.2271989-.0003.7609 3.374-.0009c.6836073 0 1.2533415-.5269875 1.321888-1.1936252l.006912-.1350748v-5.4875h2.6487c.5034404 0 .9862587-.1999887 1.342235-.555965.3559684-.3559685.555965-.8387836.555965-1.342135 0-.5034404-.1999887-.9862587-.555965-1.342235-.3164234-.3164233-.7330676-.5095973-1.175331-.5486184l-.166904-.0073466h-2.6487v-5.48747c0-.6875945-.5312955-1.26013223-1.2015872-1.32511038l-.1272128-.00613962h-5.4887v-2.6475c0-.50326472-.1999057-.98589232-.5557557-1.34172562-.3558859-.35586089-.8385119-.55577438-1.3417443-.55577438z" />
```
Comment (suevalov) [inline packages/forma-36-react-components/src/components/Icon/svg/Puzzle.tsx:6]: Is it optimized with SVGO?

Comment (suevalov) [reply packages/forma-36-react-components/src/components/Icon/svg/Puzzle.tsx:6]: Please, update the icon also in `forma-v4` branch.

Comment (Spring3) [reply packages/forma-36-react-components/src/components/Icon/svg/Puzzle.tsx:6]: I would ask that to @fabe as I wasn't creating this svg

Comment (Spring3) [reply packages/forma-36-react-components/src/components/Icon/svg/Puzzle.tsx:6]: Looks unoptimised

Comment (Spring3) [reply packages/forma-36-react-components/src/components/Icon/svg/Puzzle.tsx:6]: I'll replace it with the optimised version in a minute

**Review by suevalov — APPROVED** [review]

**Comment by Spring3** [comment]
I accidentally merged v4 into this branch

**Pull request closed without merging**